CONTRIBUTING.md

```diff
@@ -128,12 +128,24 @@ Unsure where to begin contributing to CloudQuery? You can start by looking throu
 
 If you don't see any issues that you think you can help with reach out to the community on Discord and we would be happy to work with you!
 
-
-
 #### Local Development
 
 CloudQuery has the ability to be run locally with a corresponding local postgres database. To get it up and running follow the following instructions:
 * [Building and Running the Provider Locally](../docs/index.md)
 * [Connecting to a database](https://docs.cloudquery.io/docs/getting-started#spawn-or-connect-to-a-database)
 * [Debugging a Provider](https://docs.cloudquery.io/docs/developers/debugging)
-* [Developing a New Provider](https://docs.cloudquery.io/docs/developers/developing-new-provider)
\ No newline at end of file
+* [Developing a New Provider](https://docs.cloudquery.io/docs/developers/developing-new-provider)
+
+#### Commit Messages
+
+We make use of the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/) for pull request titles. This allows us to categorize contributions and automate versioning for releases. Pull request titles should start with one of the prefixes specified in the table below:
+
+| Title      | Message | Action |
+| ----------- | ----------- |----------- |
+| `chore: <Message>`      |  `<String>`       | patch release|
+| `fix: <Message>`      |  `<String>`      | patch release|
+| `feat: <Message>`      |  `<String>`       | minor release|
+| `refactor: <Message>`      |  `<String>`       | patch release|
+| `test: <Message>`      |  `<String>`       | patch release|
+
+Additional context can be provided in parentheses, e.g. `fix(docs)`. Breaking changes should be suffixed with `!`, e.g. `feat!: drop support for X`.
```

Comment:
[non-blocking] Might worth mentioning a breaking change will always result in a minor release.

Reply:
Updated, please take another look :bow: